docs/user/tutorial/index.rst

```diff
@@ -633,30 +597,28 @@ by clicking on the :guilabel:`Download all data` button.
 Where to go from here
 ---------------------
 
-This is the end of the tutorial. You started by forking a GitHub repository
-and importing it on Read the Docs, building its HTML documentation,
-and then went through a series of steps to customize the build process,
-tweak the project configuration, and add new versions.
+This is the end of the tutorial. You have accomplished a lot:
+
+#. Forked a GitHub repository.
+#. Connected it to Read the Docs.
+#. Built its HTML documentation.
+#. Customized the build process.
+#. Added new documentation versions.
+#. Browsed the project analytics.
+
+Nice work! 
 
-Here you have some resources to continue learning about documentation
+Here are some resources to help you continue learning about documentation
 and Read the Docs:
 
-- You can learn more about the functionality of the platform
-  by going over our :doc:`features </reference/features>` page.
-- To make the most of the documentation generators that are supported,
-  you can read the :doc:`Sphinx tutorial <sphinx:tutorial/index>`
-  or the `MkDocs User Guide <https://www.mkdocs.org/user-guide/>`_.
-- Display example projects and read the source code in :doc:`/examples`.
-- Whether you are a documentation author, a project administrator, a developer, or a designer,
-  you can follow our how-to guides that cover specific tasks,
-  available under :doc:`/guides/index`.
-- For private project support and other enterprise features,
-  you can use :doc:`our commercial service </commercial/index>`
-  (and if in doubt, check out :doc:`/choosing-a-site`).
-- Do you want to join a global community of fellow `documentarians <writethedocs:documentarians>`?
-  Check out `Write the Docs <https://www.writethedocs.org/>`_ and
+- Learn more about the platform :doc:`features </reference/features>`.
+- Learn about other supported documentation generators in the :doc:`Sphinx tutorial <sphinx:tutorial/index>` or the `MkDocs User Guide <https://www.mkdocs.org/user-guide/>`_.
```

Comment:
I suppose this could be the Mkdocs tutorial now instead of a link to the user guide. That is: https://docs.readthedocs.io/en/stable/intro/getting-started-with-mkdocs.html

Reply:
Yep. I'll probably catch that when I hit the tools refactor.